client/src/pages/CheckoutPage/sections/ThankYou/ThankYou.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+import { OrderDescription, H2, OrderDetails, Address } from "./ThankYou.style";
+
+class ThankYou extends Component {
+  render() {
+    return (
+      <React.Fragment>
+        <h1>Thank you for your order</h1>
+        <OrderDescription>
+          <p>
+            Hi Paul, we’ve received your order #1223003 and we are passing it
```

Comment:
is that for the dummy data? because it should be 
"hi props.username ... props.meal"

Reply:
It's a static place holder right now as I'm not sure what you data you guys are holding in the state. We'll update this once we sync with the login / register

Reply:
alright)